scrap.py

```diff
@@ -0,0 +1,68 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+
+class Scrapping():
+    
+    def __init__(self):
+      self.url = 'https://simpsonizados.me/serie/los-simpson/'
+      self.__db = 'simpson_db.json'
+
+    def get_data(self):
+      req = requests.get(self.url)
+      page = req.text
+      soup = BeautifulSoup(page, 'lxml')
+      all_episodes = soup.find_all('ul', class_='episodios')
+      result = []
+      temp = 1
```

Comment:
Renombrar a `season`, trata de utilizar lo menos posible las abreviaciones. Siempre es preferible ser verboso y explícito que implícito.

Con la corrección del `enumerate` explicada debajo, esta variable no sería necesaria :)

Reply:
al final el enumerate no era necesario porque lo que tenia en las variables, temp y ep_count lo podia sacar directamente del for loop que esta realizando porque estaba loopeando con el range

Reply:
genial :)